packages/core/src/utils/CoreHelperUtil.ts

```diff
@@ -54,6 +54,16 @@ export const CoreHelperUtil = {
     return caipAddress.split(':')[2]
   },
 
+  getEvmChainId(caipNetworkId?: `${string}:${string}`) {
```

Comment:
There should already be a function for this? Could you check `common/NetworkUtil`'s `caipNetworkIdToNumber` function?